app/routes/__index/certificate/index.tsx

```diff
@@ -85,6 +77,25 @@ export default function CertificateIndexRoute() {
     certificate?.status === 'pending' ? 15_000 : null
   );
 
+  function formatDate(val: Date): string {
+    let date = val.toLocaleDateString('en-US', {
+      month: 'short',
+      day: '2-digit',
+      year: 'numeric',
+    });
+
+    return date;
+  }
+
+  const checkIfRenewValid = useMemo((): boolean => {
```

Comment:
The `valid` and `check` are not the best here.
`Check` indicates that this is a function, while it is not
`valid` indicates validity, while we are checking if we should allow the user to renew or not

Please rename this to `canRenew`

Reply:
Sure sounds reasonable